lib/conduit_mqtt.ex

```diff
@@ -1,7 +1,8 @@
 defmodule ConduitMQTT do
   @moduledoc """
   MQTT adapter for Conduit.
-  #TODO: put docs for connection_opts
+   * `connection_opts` - Connection opts for the adapter. For example: `[server: {Tortoise.Transport.Tcp, host: 'localhost', port: 1883}]`
+  For the full set of connection_opts, see the docs for underlying library [Tortiose](https://hexdocs.pm/tortoise/connecting_to_a_mqtt_broker.html#connection-handler).
```

Comment:
Mention that we override the handler and anything else where it doesn't matter what they set.